ISSUE_TEMPLATE.md

```diff
@@ -14,6 +14,21 @@ Large features typically go through three stages: [Alpha, Beta, and Stable](http
 Each stage requires various approvals from various teams.  Features require several releases
 to reach Stable.
 
+The following items have to be filled before the features freeze:
+
+- Primary contact (assignee);
+- SIG, responsible for the feature;
+- Proposal link - do the public design proposal in the community repo;
+- Reviewer(s) - recommend having 2+ reviewers (at least one from code-area OWNERS file) agreed to review. Reviewers from multiple companies preferred;
+- Approver (for LGTM, likely from SIG to which feature belongs);
+- Target - Alpha / beta / stable;
+- Timeline / status;
+
+Before the coding freeze:
+
+- Release note - a short sentence, describing the feature;
+- User documentation PR link (to http://github.com/kubernetes/kubernetes.github.io)
```

Comment:
@bgrant0607 not sure that have to do it in this way, but the question with docs has to be solved - we felt some difficulties to have well-prepared docs for the release earlier.